src/Utils/interfaces.tsx

```diff
@@ -0,0 +1,28 @@
+interface user {
```

Comment:
```suggestion
interface User {
```
nome de classe e interface costumamos nomear como [PascalCase](https://celsokitamura.com.br/pascal-case-e-camel-case/)